cip/CIP2015-09-16-public-type-system-type-annotation.adoc

```diff
@@ -0,0 +1,409 @@
+= CIP2015-06-16 - Public Type System and Type Annotations
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Stefan Plantikow <stefan.plantikow@neotechnology.com>
+*Author:* Nigel Small <nigel.small@neotechnology.com>
+
+[abstract]
+.Abstract
+
+This CIP specifies the Public Type System (PTS) of Cypher and introduces literal syntax for writing down types as well as new syntax for type annotations. The proposed type annotation syntax does not change the Cypher grammar right now but rather is intended as a template to be used by future CIPs. This CIP also informatively discusses future syntax that might use the introduced literal type syntax.
```

Comment:
Why do we need to have an abbreviation for the public type system? I feel that this is unnecessary, and that "PTS" in particular is not a very good abbreviation, since it makes me think of "Post Traumatic Stress" before I think of "Public Type System"...